src/Common/Services/LicenseServices.cs

```diff
@@ -37,26 +30,26 @@ public async Task<CPSoftwareLicense> GetLicenseByUniqueName(string licenseUnique
                     Operator= FieldOperator.GreaterThan,
                     Value = 0
                 }
-            };
+            ];
 
-        GetObjectsByFilterInput gobfiInput = new GetObjectsByFilterInput
+        GetObjectsByFilterInput gobfiInput = new()
         {
             Filter = fcCollection,
-            Type = Activator.CreateInstance<CPSoftwareLicense>()
+            Type = new CPSoftwareLicense()
         };
 
-        GetObjectsByFilterOutput gobfOutput = await gobfiInput.GetObjectsByFilterAsync(true);
-
-        if (gobfOutput.Instance.Count == 0)
+        var licenses = (await gobfiInput.GetObjectsByFilterAsync(true)).Instance.Cast<CPSoftwareLicense>();
+        if (!licenses.Any())
```

Comment:
careful here, licenses for some reason can be null if the cast fails. You should protect here to check if is also null